.github/workflows/on-release.yaml

```diff
@@ -9,9 +9,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-        with:
-          fetch-depth: "0"
-
```

Comment:
This was added recently IIRC - what was is it about?

Reply:
I removed it everywhere it's not necessary.

Instead of doing a shallow clone, it does a git clone and fetches the whole history.

Reply:
If necessary, I can extract this into another PR, but I have the impression we have more PRs than merges, so I didn't want to spam PRs, like I usually do.

Reply:
:+1: